fmriprep/utils/transforms.py

```diff
@@ -81,15 +70,8 @@ def load_ants_h5(filename: Path) -> nt.base.TransformBase:
         raise ValueError(msg)
 
     fixed_params = transform2['TransformFixedParameters'][:]
-    if not np.array_equal(fixed_params, FIXED_PARAMS):
-        msg = 'Unexpected fixed parameters\n'
-        msg += f'Expected: {FIXED_PARAMS}\n'
-        msg += f'Found: {fixed_params}'
-        if not np.array_equal(fixed_params[6:], FIXED_PARAMS[6:]):
-            raise ValueError(msg)
-        warnings.warn(msg, stacklevel=1)
 
-    shape = tuple(fixed_params[:3].astype(int))
+    shape = tuple(fixed_params[:3].astype(int)[::-1])
     warp = h['TransformGroup']['2']['TransformParameters'][:]
     warp = warp.reshape((*shape, 3)).transpose(2, 1, 0, 3)
```

Comment:
This is a really unintuitive change that deserves a comment, so I've thought it through. I've decided we actually want to modify the reshape, not the shape.

What's going on here is that we have a Fortran-ordered array, so the first index changes fastest, and the order of indices is `(component, i, j, k)`. Then we want three NIfTI volumes, which are also Fortran ordered, but that means we want to transpose to `(i, j, k, component)`. We've been treating the vector component as if it's in the right place, and everybody else is out of step. Simpler to just move the vector component to the end:

```suggestion
    shape = tuple(fixed_params[:3].astype(int))
    # ITK stores warps in Fortran-order, where the vector components change fastest
    # Nitransforms expects 3 volumes, not a volume of three-vectors, so transpose
    warp = np.reshape(transform2['TransformParameters'], (3, *shape), order='F').transpose(1, 2, 3, 0)
```

Reply:
This seems much more readable - let's try it